lib/cmsis_rtos_v1/cmsis_kernel.c

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2018 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel_structs.h>
+#include <cmsis_os.h>
+
+/**
+ * @brief Get the RTOS kernel system timer counter
+ *
+ * Returns
+ * RTOS kernel system timer as 32-bit value
+ */
+uint32_t osKernelSysTick(void)
+{
+	return k_cycle_get_32();
+}
```

Comment:
A whole source file with copyright header and two (!) include files just for a single one-line function wrapper.  Seriously? :)

Can't you fold this together with the other files?

Reply:
This will have further additions (at least 3 more functions).